client/allocrunner/taskrunner/task_runner.go

```diff
@@ -404,7 +404,7 @@ func (tr *TaskRunner) Run() {
 		case <-tr.shutdownCtx.Done():
 			return
 		case <-tr.serversContactedCh:
-			tr.logger.Trace("server contacted; unblocking waiting task")
+			tr.logger.Info("server contacted; unblocking waiting task")
```

Comment:
I'd suggest updating the log level in https://github.com/hashicorp/nomad/blob/92527c6b4ef8951d82a49367737eca6ef5e0b9ff/client/allocrunner/taskrunner/task_runner.go#L902-L903 to be consistent with this too.

Reply:
gonna leave that for now, bc this is mostly pairing with the log on 401, and updating :902 would cause more double logging.

Reply:
In that case, I suspect for logging purposes, it's more important to log that we are blocking startup of a task - when we start a task we have plenty of other log lines indicating task is starting.  either way not a big deal.